WindowState/README.md

```diff
@@ -6,104 +6,211 @@ The component provides the current window information as a Compose state, includ
+
+```kotlin
+val foldState: FoldState = FoldState.FLAT
+```
+
+Returns the state of a fold: **FLAT** or **HALF_OPENED**. If no fold is present, the returned state is flat.
+
+Based on the [state](https://developer.android.com/reference/androidx/window/layout/FoldingFeature#state()) field in [FoldingFeature](https://developer.android.com/reference/androidx/window/layout/FoldingFeature).
+
+```kotlin
+val foldIsSeparating: Boolean = false
+```
+
+Returns whether a fold should be thought of as separating the window into distinct sections.
+
+Based on the [isSeparating](https://developer.android.com/reference/androidx/window/layout/FoldingFeature#isSeparating()) field in [FoldingFeature](https://developer.android.com/reference/androidx/window/layout/FoldingFeature).
+
+```kotlin
+val foldIsOccluding: Boolean = false
+```
+
+Returns whether a fold occludes content in the window.
+
+Based on the [occlusionType](https://developer.android.com/reference/androidx/window/layout/FoldingFeature#occlusionType()) field in [FoldingFeature](https://developer.android.com/reference/androidx/window/layout/FoldingFeature).
+
+```kotlin
+val foldSizeDp: Dp
+```
+
+Returns a dp value of the thickness of the hinge of dual-screen device or the folding line of foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
 ```kotlin
-val foldablePaneWidth: Int
+val foldablePane1SizeDp: Size
 ```
-Returns a pixel value of the width of a single pane of the dual-screen or foldable device when the device is in the dual-screen mode. If the device is in the single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+Returns the dp size of the primary pane of the dual-screen or foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+The primary pane is always the top pane and the left/right pane, depending on the local language layout direction.
 
 ```kotlin
-val foldSize: Int
+val foldablePane2SizeDp: Size
 ```
-Returns a pixel value of the width of the hinge of dual-screen device or the folding line of foldable device when the device is in the dual-screen mode. If the device is in the single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+Returns the dp size of the secondary pane of the dual-screen or foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+The secondary pane is always the bottom pane and the left/right pane, depending on the local language layout direction.
+
+```kotlin
+val pane1SizeDp(pane1Weight: Float = 0.5f): Size
```

Comment:
Can we mention that we recommend to use `pane1SizeDp` instead of `foldablePane1SizeDp`, unless the large-screen enhancement is preferred to be ignored?

Reply:
I updated the description, let me know if you think that's better or if it needs more emphasis!